### PR TITLE
Updates supertokens-node dependency version

### PR DESCRIPTION
**Pull request by rishabhpoddar**
We update the dependency version to v7.0.0 and make a few code changes to work with the new version

### CODE REVIEW

File: examples/with-supertokens/package.json [hunk 1]
```diff
@@ -10,9 +10,10 @@
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "supertokens-auth-react": "^0.15.0",
-    "supertokens-node": "^6.0.0"
+    "supertokens-node": "^7.0.0"
   },
+  "license": "MIT",
```
Comment (leerob) [inline examples/with-supertokens/package.json:15]:
```suggestion
```

**Comment by rishabhpoddar** [comment]
@leerob any updates on this please? Thanks.

**Review by leerob — DISMISSED** [review]

**Comment by leerob** [comment]
Looks like linting is failing - could you fix that please?

**Review by leerob — APPROVED** [review]

**Comment by rishabhpoddar** [comment]
thanks @leerob

**Pull request merged**